edge/pkg/edged/server/server.go

```diff
@@ -14,7 +14,7 @@ import (
 //constants to define server address
 const (
 	ServerAddr = "127.0.0.1"
-	ServerPort = "10255"
+	ServerPort = "10350"
```

Comment:
why?

Reply:
This is to support both the logs and exec feature 。 #1477

Reply:
Yes, we can defined as config options later along with cloud side.